test/system/react_components/maintaining_iterations_summary_table_test.rb

```diff
@@ -0,0 +1,17 @@
+require "application_system_test_case"
+
+class MaintainingIterationsSummaryTableTest < ApplicationSystemTestCase
+  test "visiting the index" do
+    solution = create :concept_solution
+    iteration = create :iteration, solution: solution
+
+    visit maintaining_iterations_summary_table_url()
+    sleep(0.1) # Set the websocket connection
+
+    iteration.tests_passed!
+    iteration.broadcast!
+
+    sleep(0.1) # Wait for websocket to broadcast
+    assert_text "#{iteration.id}"
+  end
+end
```

Comment:
So, general comment on this file. I'd like to test components independently of the pages they are on. The test routes (`app/views/test/...`) aims to do this. So in these system component tests, I'd rather us call the test route and test that component in isolation.

We can then have a second test later that tests the component in the context of the actual eventual page it's on.

These would sit in `test/system/maintaining`. 

Does that make sense?